pennylane/qchem/observable_hf.py

```diff
@@ -142,7 +142,7 @@ def qubit_observable(o_ferm, cutoff=1.0e-12):
     return simplify(qml.Hamiltonian(coeffs, ops), cutoff=cutoff)
 
 
-def jordan_wigner(op, notation="physicist"):
+def jordan_wigner(op: list, notation="physicist"):  # pylint:disable=too-many-branches
```

Comment:
If the old jordan_wigner function was top level accessible, then I think it should be fine to just remove this completely! There may be some tests / parts of pennylane where we import it directly like `from pennylane.qchem import jordan_wigner`, but you can just find and replace all such instances with `qml.jordan_wigner`

Reply:
It wasn't, it was only accessible as `qml.qchem.jordan_wigner`